packages/mc-scripts/src/cli.ts

```diff
@@ -197,6 +198,43 @@ async function run() {
       }
     );
 
+  // Command: deployment-previews:push
+  const usageDeploymentPreviewsPush =
+    'Creates or updates a deployment preview for the customization application.';
+  cli
+    .command('deployment-previews:push', usageDeploymentPreviewsPush)
```

Comment:
Thinking about the naming, I find `push` slightly confusing considering what it does.

I suggest something like `set` (since we're doing an upsert). Vercel for instance [uses that](https://vercel.com/docs/cli/alias#usage) as well `vercel alias set`.

```suggestion
    .command('deployment-previews:set', usageDeploymentPreviewsPush)
```

This can then scale with other commands:

```
deployment-previews:set
deployment-previews:remove
deployment-previews:list
```

Reply:
I personally prefer `push` as a more meaningful word (maybe only from my point of view) since that name does not imply how internally the logic will work but if you prefer a different naming, I'll just adjust things accordingly.

Reply:
I don't have a strong opinion, it was just my feeling as a consumer of the CLI.

Let's wait for the feedback from the others.

Reply:
I can only share my mental model for `push` and `set`. I would prefer `set` if you `set` (update or create) a value singular entity. I would use `push` (or likely more `add`) if we add to a list of things. I would then exact that the addition to that bag of things as an effect or later I have to `mark` or `set` the added item to e.g. be the primary of something.

Reply:
Thanks for your feedback, @tdeekens 

I guess that in this context you would prefer `add` as the name of the command?

Bear in mind what we're actually doing is adding a new deployment preview link to the list of deployment preview links of a Custom Application. If the alias already exist, we update the current preview link configuration url.

Reply:
Yes, I'd have chosen `add` then. Still, no strong opinions here. It's a suggestion, nothing more.

Reply:
Thanks!
We have several suggestions but we need to come to an agreement.

So far we have `push`, `set` and `add`.

How we should decide on what to use?

Reply:
@kark @Rhotimee Please can you also share your feedback here?

Reply:
Thank you for pinging me. First and foremost, this is a great improvement 💪 many thanks for the efforts! 🙇‍♂️

I believe that `push`, `add`, and `set` all work well in this context. However, I have a slight preference for `set`. To me, it clearly suggests intent of either creation or updating of a deployment preview config.

Reply:
Suggestion for a slight different option maybe: have "CRUD-like" commands like `create`, `update`, `remove`, etc.
Then it becomes clear and more explicit what the command does. This means of course that there won't be any upsert behavior.

Reply:
I also prefer `set` because it indicates an upsert operation.

Reply:
Ok, thank you all for the feedback.

We are two who prefer `add` but three who prefer `set` so I'll be changing the naming of the command.

Reply:
Command renamed in this commit: aa8ac4684f43bce4c5332169e38bd039964e6212